### ISSUE TITLE
DatabaseException: "orig" parameter not provided to the base exception class

### ISSUE THREAD

**Issue by hyst329** [xzkostyan/clickhouse-sqlalchemy#13]
At the moment, `DatabaseException` constructor now looks like this:
```python
class DatabaseException(Exception):
    def __init__(self, orig):
        self.orig = orig
        super(DatabaseException, self).__init__()
```
It can cause problems when `DatabaseException` is being thrown from a `multiprocessing`s process (e.g. while working with big data in ClickHouse in parallel), `ForkingPickler` complaining about the missing `orig` parameter. So I suggest to add `orig` into the super constructor, because `orig` is non-optional:
```python
class DatabaseException(Exception):
    def __init__(self, orig):
        self.orig = orig
        super(DatabaseException, self).__init__(orig)
```

**Comment by xzkostyan** [comment]
@jhirniak seems legit. Thanks.

**Comment by xzkostyan** [comment]
Fixed in version 0.0.8.
